packages/core/videoClient.ts

```diff
@@ -384,6 +384,30 @@ const checkIfRoomNameMatchesInRecording = async (roomName: string, recordingId:
   return videoAdapter?.checkIfRoomNameMatchesInRecording?.(roomName, recordingId);
 };
 
+const getWebhooks = async () => {
```

Comment:
Right now this is only working for daily. So, we should change it's name maybe for now atleast.

Ideally an app specific code shouldn't exist here.